docs/examples.md

```diff
@@ -0,0 +1,67 @@
+# Examples & Use cases
+
+## Official examples
+
+[The Puppeteer repository](https://github.com/puppeteer/puppeteer/tree/main/examples) includes a small number of examples maintained by the Puppeteer team.
+
+Follow the instructions in the README to run the examples, covering use cases like creating PDFs from websites, creating screenshots or intercepting requests.
+
+## Example suite
+
+Find a set of unstructured examples in Puppeteer's dedicated [example repository](https://github.com/puppeteer/examples).
+
+This suite is a collection of examples that has been growing over time and covers various use cases like forwarding events
+from your Puppeteer process to the browser, interacting with elements and running CDP commands.
+
+## Other projects, articles and demos
+
+See the following list for use cases and examples from categories like Rendering, Web scraping and Testing.
+
+### Rendering and web scraping
```

Comment:
Many projects/examples here seem somewhat outdated. Do you think we should keep referring to them or try to update them? Or perhaps we should discover some newer maintained examples? WDYT?

Reply:
Yep they definitely are, pondered about the same when going through them... I think trying to update the list to more current/maintained ones would be good.

However I could also imagine keeping some of the older ones around to inspire beyond usual tooling, Puppetron for example is a fun one - although it's arguably outdated too.

I'll document thoughts and ideas more properly next week and share with you :) while I was collecting everything today I just thought why not get at least this landed quickly.